proposals/p2022.md

```diff
@@ -0,0 +1,276 @@
+# Unused Pattern Bindings (Unused Function Parameters)
+
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+[Pull request](https://github.com/carbon-language/carbon-lang/pull/2022
+
+<!-- toc -->
+
+## Table of contents
+
+-   [Problem](#problem)
+-   [Background](#background)
+-   [Proposal](#proposal)
+-   [Details](#details)
+    -   [The behavior of `unused` name bindings](#the-behavior-of-unused-name-bindings)
+    -   [Examples](#examples)
+-   [Rationale](#rationale)
+-   [Alternatives considered](#alternatives-considered)
+    -   [Commented Names](#commented-names)
+    -   [Only short form support with `_`](#only-short-form-support-with-_)
+    -   [Anonymous, Named Identifiers](#anonymous-named-identifiers)
+
+<!-- tocstop -->
+
```

Comment:
We'd like to include an abstract section as of  #1895 . It can match the summary in GitHub.

```suggestion
## Abstract

This proposal specifies how unused pattern bindings are written in the Carbon
programming language. This is a more general problem statement of "how do
users specify unused function parameters" as function parameter declarations
are a more specific form of pattern.

Related issue:
[#1996](https://github.com/carbon-language/carbon-lang/issues/1996).
```

Reply:
Done